xml/art_migration.xml

```diff
@@ -0,0 +1,692 @@
+    </para>
+    </listitem>
+    <listitem>
+     <para>
+      System clock synchronization across cloud nodes?
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      HA capacity of &contrnode;s (L3/DHCP agent)?
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      HA capacity of &contrnode;s (&swift;)?
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Data model and &o_blockstore; back-end compatibility (no VSA or Ceph)?
+     </para>
+    </listitem>
+  </itemizedlist>
+</step>
+<step>
+  <para>
+   Identify a candidate seed &contrnode; in the Primary control plane. This
+   node must meet the following requirements:
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     Networking connectivity is equivalent to that of the existing deployer.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     It is a member of the database cluster (running the MySQL service).
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+      It is not actively hosting any singleton &ostack; services
+      (<systemitem>cinder-volume</systemitem> or
+      <systemitem>nova-consoleauth</systemitem>). Test for empty output with
+      the following command:
+    </para>
+    <screen>(each controller)> ps -ef | egrep 'cinder-volume|nova-consoleauth'</screen>
+   </listitem>
+  </itemizedlist>
+</step>
+<step>
+  <para>
+   Prepare the cloud for migration:
+  </para>
+  <itemizedlist>
+  <listitem>
+   <para>
+    Ensure that the data model enables deployer behavior by the new seed
+    node.
+   <remark>taroth 2018_02-27: what is this supposed to mean?</remark>
+   </para>
+   <itemizedlist>
+    <listitem>
+     <para>
+      In the <filename>disks_*.yml</filename> file used by your server role
+      that includes monitoring services, replace the existing <literal>Vertica</literal>
+      partition definition with two <literal>Cassandra</literal> partitions:
+     </para>
+   <screen>- name: cassandra
+            size: (whatever size was used originally by vertica partition, less 1%)
+            mount: /var/cassandra/data
+            fstype: ext4
+            mkfs-opts: -O large_file
+            consumer:
+              name: vertica
+          - name: cassandra_log
+            size: 1%
+            mount: /var/cassandra/commitlog
+            fstype: ext4
+            mkfs-opts: -O large_file
+   </screen>
+    </listitem>
+    <listitem>
+     <para>
+      Edit
+      <filename>~/helion/my_cloud/definition/data/control_plane.yml</filename>
+      and identify the cluster definition that already includes an entry for
+      <filename>mysql</filename> in its service-components.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Ensure that <systemitem>lifecycle-manager</systemitem> is included in the same
+      cluster's service-components list. If it was necessary to add this as a
+      new entry, it will also need to be removed from its original position in
+      the deployer.
```

Comment:
Does `service-components` need to go into a literal?

Rephrasing proposal: "Ensure that <systemitem>lifecycle-manager</systemitem> is included in the same cluster's service-components list. If it is not included, add it to and also remove it from its original position in the deployer [definition][?]."